Tests/Indicators/AlphaIndicatorTests.cs

```diff
@@ -0,0 +1,215 @@
+        {
+            var indicator = CreateIndicator();
+            var firstVolumeRenkoConsolidator = new VolumeRenkoConsolidator(1000000);
+            var secondVolumeRenkoConsolidator = new VolumeRenkoConsolidator(1000000000);
+            firstVolumeRenkoConsolidator.DataConsolidated += (sender, renkoBar) =>
+            {
+                Assert.DoesNotThrow(() => indicator.Update(renkoBar));
+            };
+
+            secondVolumeRenkoConsolidator.DataConsolidated += (sender, renkoBar) =>
+            {
+                Assert.DoesNotThrow(() => indicator.Update(renkoBar));
+            };
+
+            foreach (var parts in GetCsvFileStream(TestFileName))
+            {
+                var tradebar = parts.GetTradeBar();
+                if (tradebar.Symbol.Value == "AMZN")
+                {
+                    firstVolumeRenkoConsolidator.Update(tradebar);
+                }
+                else
+                {
+                    secondVolumeRenkoConsolidator.Update(tradebar);
+                }
+            }
+
+            Assert.IsTrue(indicator.IsReady);
+            Assert.AreNotEqual(0, indicator.Samples);
+            firstVolumeRenkoConsolidator.Dispose();
+            secondVolumeRenkoConsolidator.Dispose();
+        }
+
+
+        [Test]
+        public void AcceptsQuoteBarsAsInput()
+        {
+            var indicator = new Alpha("testAlphaIndicator", Symbols.IBM, Symbols.SPY, 5);
+
+            for (var i = 10; i > 0; i--)
+            {
+                indicator.Update(new QuoteBar { Symbol = Symbols.IBM, Ask = new Bar(1, 2, 1, 500), Bid = new Bar(1, 2, 1, 500), Time = _reference.AddDays(1 + i) });
+                indicator.Update(new QuoteBar { Symbol = Symbols.SPY, Ask = new Bar(1, 2, 1, 500), Time = _reference.AddDays(1 + i) });
+            }
+
+            Assert.AreEqual(2, indicator.Samples);
+        }
+
+        [Test]
+        public void EqualAlphaValue()
+        {
+            int period = 5;
+            var indicator = new Alpha("testAlphaIndicator", Symbols.AAPL, Symbols.SPX, period);
+
+            for (int i = 0; i <= period; i++)
+            {
+                indicator.Update(new TradeBar() { Symbol = Symbols.AAPL, Low = 1, High = 2, Volume = 100, Close = 100 + i, Time = _reference.AddDays(1 + i) });
+
+                Assert.AreEqual(0.0, (double)indicator.Current.Value, 0.0000000001);
+
+                indicator.Update(new TradeBar() { Symbol = Symbols.SPX, Low = 1, High = 2, Volume = 100, Close = 200 + i * 3, Time = _reference.AddDays(1 + i) });
+
+                if (i < period)
+                {
+                    Assert.AreEqual(0.0, (double)indicator.Current.Value, 0.0000000001);
+                }
+                else
+                {
+                    Assert.AreEqual(0.0032062252, (double)indicator.Current.Value, 0.0000000001);
+                }
+
+            }
+        }
+
+        [Test]
+        public void NotEqualAlphaValue()
```

Comment:
What is this test asserting?

Reply:
I've copied from tests for Beta, but you're right... I'll change it to something valuable.